src/user-event/press/__tests__/press.test.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Text } from 'react-native';
+import { createEventLogger } from '../../../test-utils';
+import { render } from '../../..';
+import { userEvent } from '../..';
+
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+describe('user.press()', () => {
+  it('dispatches required events on Text', async () => {
+    // Required for touch events which contain timestamp
+    jest.spyOn(Date, 'now').mockReturnValue(100100100100);
```

Comment:
Do we really want the events builded to use Date.now? Having to mock it in tests, especially when not using real timers does not seem ideal. Is it because they're also used in userEvent implementation? If that's the case I think default events used for the implementation and mocks should be two distinct things, although they could share the same interface

Reply:
We could go with zero here, will simplify the code indeed. 

The reason I've used `Date.now()` was that of all the values we have in that event, this is the single one where would code provide some meaningful value ;-)

Reply:
Changed to use `timestamp: 0`.